content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,460 @@ linkTitle: "Workload Manager"
+
+### Submit a service order via the Marketplace
+
+User can request access to the service submitting a service-order to use the EGI
+HTC service directly from the EOSC or the EGI Marketplace:
+
+EOSC Marketplace:
+[https://marketplace.eosc-portal.eu/services/egi-workload-manager](https://marketplace.eosc-portal.eu/services/egi-workload-manager)
+
+EGI Marketplace:
+[https://marketplace.egi.eu/compute/73-workload-manager.html](https://marketplace.egi.eu/compute/73-workload-manager.html)
+
+Service orders are usually handled within 5 working days by the EGI User Support
+Team on shift.
+
+### Before to start
+
+#### Apply for your user credentials
+
+DIRAC uses X.509 certificates to identify and authenticate users.  These
+certificates are delivered to each individual by trusted certification
+authorities.  If you have a personal certificate issued by a EUGridPMA-certified
+authority you can use it for this tutorial.  Otherwise, IHEP certification
+authority can deliver one for you.  Click on the link Online for User Request on
+this page and follow the instructions.  Your certificate may take a few days to
+be delivered, so please ask for your certificate well in advance and in any
+case, before the tutorial starts.
+
+#### Export your credentials from your browser
+
+Your personal certificate is usually delivered to you via a web site and is
+automatically loaded in your browser.  You need to export it from the browser
+and put it in the appropriate format for DIRAC to use.  This is a one-time
+operation.  Please follow the instructions in this document to export your
+certificate to a local file and named $HOME/MyCertificate.
+
+#### Install your credentials
+
+You can user the commands below in any Unix machine to get your user credentials
+in the format and location expected by DIRAC:
+
+```
+$ mkdir -p $HOME/.globus
+$ chmod 700 $HOME/.globus
+$ openssl pkcs12 -in $HOME/MyCertificate \ -clcerts -nokeys -out $HOME/.globus/usercert.pem
+$ openssl pkcs12 -in  $HOME/MyCertificate\ -nocerts -out $HOME/.globus/userkey.pem
+$ chmod 600 $HOME/.globus/usercert.pem $HOME/.globus/userkey.pem ```
+```
+
+#### Send your certificate's subject to the DIRAC team
+
+In order to configure the DIRAC server so that you get registered as a user, the
+team needs to know your certificate's subject.
+
+Please use the command below on any Unix machine and send its output to Andrei
+Tsaregorodtsev (atsareg`<AT>`in2p3`<DOT>`fr):
+
+```sh
+$ openssl x509 -in $HOME/.globus/usercert.pem -subject -noout
+```
+
+### The EGI Workload Manager Web Portal
+
+To access the EGI Workload Manager open a web browser to:
+[https://dirac.egi.eu/DIRAC/](https://dirac.egi.eu/DIRAC/)
+
+![Fig. 2 - The EGI Workload Manager service Web portal](./figure_2.png)
+
+Fig. 2 - The EGI Workload Manager service Web portal
+
+If you are a new user, you can see the welcome page where you can find links to
+user documentations.  VO options: you can switch to different VOs that you have
```

Comment:
It may be easier fo readers if things like `VO options:` and `Log in option` could be emphasised.

Reply:
584aa231a0a619522fe83adb666ed1aaa728fee5